pallet/ecdsa-authority/Cargo.toml

```diff
@@ -13,7 +13,6 @@ ethabi     = { version = "17.0", default-features = false }
 scale-info = { workspace = true }
 # crates.io optional
 # https://github.com/paritytech/libsecp256k1/issues/134
-# array-bytes  = { workspace = true, optional = true }
```

Comment:
Nope. This optional is unused now that we can comment on it.

It does relate to the https://github.com/paritytech/libsecp256k1/issues/134.

So, please don't remove it.